### ISSUE TITLE
🐛 FIX: regex of `sphinx-copybutton` not working for MyST

### ISSUE THREAD

**Issue by mbercx** [aiidateam/aiida-tutorials#341]
It seems the regex for the `sphinx-copybutton` is no longer working as intended. For example when using the copy button for the following snippet:

![Screenshot 2021-06-08 at 19 07 30](https://user-images.githubusercontent.com/25607960/121228213-c752c880-c88c-11eb-8c89-3221f19f6836.png)

It copies _all_ the content:

```console
$ verdi group show tutorial_pbesol
-----------------  ----------------
Group label        tutorial_pbesol
Group type_string  core
Group description  <no description>
-----------------  ----------------
# Nodes:
PK    Type         Created
----  -----------  -----------------
 380  CalcJobNode  2078D:17h:46m ago
1273  CalcJobNode  2078D:18h:03m ago
...
```

Instead of the desired `verdi group show tutorial_pbesol`. Corresponding MyST Markdown:

```md
```{code-block} console

$ verdi group show tutorial_pbesol
-----------------  ----------------
Group label        tutorial_pbesol
Group type_string  core
Group description  <no description>
-----------------  ----------------
# Nodes:
PK    Type         Created
----  -----------  -----------------
 380  CalcJobNode  2078D:17h:46m ago
1273  CalcJobNode  2078D:18h:03m ago
...

```#
```

@chrisjsewell do you know an easy fix for this? I know we'll eventually want to move to making these all executable notebooks, but there might still be some parts where we want to use the copy button. Moreover, I'm not sure if we'll be able to adapt all the content in time for the tutorial.

**Comment by chrisjsewell** [comment]
> @chrisjsewell do you know an easy fix for this

I've been "allowing it" to go on in the background at the moment lol, but I'm seeing a bunch of notifications about PRs and stuff on sphinx-copybutton so I will let you know when I circle round to checking what dameage they've done 😅

**Comment by mbercx** [comment]
Haha, ok. You give em hell! 🤣

**Comment by mbercx** [comment]
@chrisjsewell did you manage to have a look at this? If we can't use the regex to copy only the input content of the `code-block`s, it might be best to remove the copy button for now.

**Comment by mbercx** [comment]
This was fixed by #362

Note that empty lines are also copied as end-of-line characters, so watch out for those. Might raise an issue on `sphinx-copybutton`, because I'm not sure that's the intended behaviour.
